apps/condo/domains/billing/components/ReportMessage.tsx

```diff
@@ -0,0 +1,48 @@
+import { useIntl } from '@core/next/intl'
+import { useMemo } from 'react'
+import { Typography } from 'antd'
+import { colors } from '@condo/domains/common/constants/style'
+import React from 'react'
+
+type LastReportType = {
+    totalReceipts: number
+    finishTime: string
+}
+
+interface IReportMessageProps {
+    lastReport: LastReportType
+}
+
+export const ReportMessage: React.FC<IReportMessageProps> = ({ lastReport }) => {
+    const intl = useIntl()
+    return useMemo(() => {
+        const DataWasUploadedSuffixMessage = intl.formatMessage({ id: 'DataWasUploadedOnSuffix' })
+        const DataWasUploadedPrefixMessage = intl.formatMessage({ id: 'DataWasUploadedOnPrefix' })
+        const NoReportMessage = intl.formatMessage({ id: 'NoReceiptsLoaded' })
+        if (!lastReport) {
+            return (
+                <Typography.Text style={{ color: colors.orange[5] }}>
```

Comment:
css hack detected use type="warning"

Reply:
Our Figma references uses different color here.
<img width="285" alt="image" src="https://user-images.githubusercontent.com/50069872/131632266-ecd689b1-c459-4f53-b547-aadab4bd0f71.png">
<img width="222" alt="image" src="https://user-images.githubusercontent.com/50069872/131632307-8fa26592-1b05-40d1-8258-40842169b742.png">

Reply:
Resolved 👍 
Change global styles for type = ['warning', 'success', 'danger'], depending on our design system